VR Maze/Assets/Scripts/GameTimer.cs

```diff
@@ -10,7 +10,7 @@ public class GameTimer : MonoBehaviour
 {
     public Canvas gameOver;
 
-    public float MaxGameTime = 15; //equivalent of 5 minutes
+    public float MaxGameTime = 300; //equivalent of 5 minutes
```

Comment:
Does this not massively extend game time? I don't see this being referenced anywhere else in the changes.